lib/gems/pending/appliance_console/cli.rb

```diff
@@ -25,6 +27,8 @@ module ApplianceConsole
   class Cli
     attr_accessor :options
 
+    NETWORK_INTERFACE = "eth0".freeze
```

Comment:
I see why you would need this, but I would rather change the `DatabaseReplicationStandby` class so that this was a parameter rather than have to define a constant in any class that uses it.